koku/masu/api/trino.py

```diff
@@ -32,6 +32,7 @@ def trino_query(request):
         data = request.data
         query = data.get("query")
         schema_name = data.get("schema")
+        query = query.removesuffix(";")
```

Comment:
Move this after the `if query is None` check.

Reply:
Great catch @samdoran